src/tokens.md

```diff
@@ -464,7 +441,7 @@ let horse = example.0b10;  // ERROR no field named `0b10`
 > **<sup>Lexer</sup>**\
 > FLOAT_LITERAL :\
 > &nbsp;&nbsp; &nbsp;&nbsp; DEC_LITERAL `.`
->   _(not immediately followed by `.`, `_` or an [identifier]_)\
+>   _(not immediately followed by `.`, `_` or an [identifier] or [keyword][keywords]_)\
```

Comment:
To be pedantic here, I think it is an identifier, raw identifier, or keyword.  We don't have have anything to succinctly refer to that. What do you think about adding a new rule to the identifiers chapter?  Maybe something like:

> IDENTIFIER_OR_KEYWORD_OR_RAW: IDENTIFIER_OR_KEYWORD | RAW_IDENTIFIER

I...can't think of a good name for that, maybe you can improve that.  That rule can then be referenced here, and the macros-by-example chapter (where it refers to `ident` in the Metavariables section).

Reply:
Ah, it could be a byte literal or a raw string literal too.

I think it will be best to switch to just talking about the following character directly, rather than how the following input would be tokenised (so matching the implementation more closely).